docs/releases.md

```diff
@@ -1,12 +1,14 @@
+# Releases
+
 This document describes how the release flow looks like and how to create a bundle release.
 
 ## Release process
 
-Every change in the master branch which passes all checks triggers creating a `latest` tag and release.
+Every change in the master branch which passes all checks triggers the `latest` tag and release.
```

Comment:
triggers or creates the `latest` tag? Maybe creates the `latest` tag and triggers a release?

Reply:
`creates the latest tag and triggers a release` option is good